package.json

```diff
@@ -23,6 +23,7 @@
     "react-component"
   ],
   "devDependencies": {
+    "babel-polyfill": ">=6.0.0",
```

Comment:
I don't think this should also moved into peerDependencies, I think `babel-polyfill` should be set when bundling scripts in their projects. BTW, I've used my fork version to build a project https://github.com/Canner/grid-breakpoint . I think the setting here works great to me.